pkg/controller/cluster/cluster_controller.go

```diff
@@ -519,6 +520,12 @@ func (c *ClusterController) manageMachineSet(cluster *clusteroperator.Cluster, m
 		return machineSet, true, err
 	}
 
+	if !apiequality.Semantic.DeepEqual(machineSet.Spec.Version, cluster.Spec.Version) {
+		glog.V(2).Infof("The cluster version of the machine set %s has changed from %v to %v", machineSet.Name, machineSet.Spec.Version, cluster.Spec.Version)
```

Comment:
This comment comes off as a bit misleading to me. It states that the cluster version of the machine set changed, when in actuality the cluster version of the cluster changed.